src/dfx-core/src/error/dfx_config.rs

```diff
@@ -58,6 +58,12 @@ pub enum GetWasmMemoryLimitError {
     GetWasmMemoryLimitFailed(String, GetCanisterConfigError),
 }
 
+#[derive(Error, Debug)]
+pub enum GetLogVisibilityError {
+    #[error("Failed to get log visibility for canister '{0}': {1}")]
+    GetLogVisibilityFailed(String, GetCanisterConfigError),
```

Comment:
```suggestion
    #[error("Failed to get log visibility for canister '{0}'")]
    GetLogVisibilityFailed(String, #[source] GetCanisterConfigError),
```

Reply:
I see GetWasmMemoryLimitError has the same problem, but the others follow this pattern